src/app/shared/shared.module.ts

```diff
@@ -21,18 +23,30 @@ const declarations = [components, directives, pipes].reduce(
     []
 );
 
+// services which will be loaded in different way
+const skipServices = ['AuthorizedHttp'];
 // Get only providers, not types or other objects
-const providers = Object.keys(services)
+let providers = Object.keys(services)
```

Comment:
const to operator pojedynczego przypisania a nie 'niemodyfikowalnosci'. Mozna tutaj bylo smialo zostawic const.